src/azul/openapi/responses.py

```diff
@@ -1,16 +1,27 @@
 from azul.openapi import (
     format_description_key,
     schema,
+    application_json,
 )
 from azul.openapi.schema import (
-    TYPE,
```

Comment:
Could you revert this and fix your PyCharm settings? The recommended PyCharm formatter settings are in the repo at the project root so you can import them. I think they specify the trailing comma. Let me know if they don't.

Reply:
My formatter settings are up to date. The issue is that the pertinent formatter setting is only triggered for _multiline_ `from` imports:

```
        <option name="FROM_IMPORT_TRAILING_COMMA_IF_MULTILINE" value="true"/>
```

I ran a test:
```
$ head -8 src/azul/openapi/params.py
from typing import (
    Union
)

from azul.openapi import (
    format_description_key,
    schema
)
$ make format
...
$ head -8 src/azul/openapi/params.py
from typing import (
    Union
)

from azul.openapi import (
    format_description_key,
    schema,  # <- comma only added here
)

Reply:
I think some team members may not use these settings, which could explain the inconsistencies. I don't catch them all. Until we have a process to enforce this, here's what *should* happen: multiline imports (including multiline imports with just a single imported symbol) should have a trailing comma after each symbol.